README.md

```diff
@@ -55,3 +55,147 @@ key and all necessary information are provided.
 ## Usage
 
 Run `astartectl` to see available commands.
+
+### Send or set data to devices
+
+For server-owned interfaces, it is possible to send or set data via astartectl.
+To do so, you can use:
+- astartectl appengine devices send-data (deprecated)
+- astartectl appengine devices publish-datastream
+- astartectl appengine devices set-property
+
+Run the appropriate command to see a short description, command suggestions and flags.
+Generally speaking, one of those could be written like 
+```
+ astartectl appengine devices publish-datastream 2TBn-jNESuuHamE2Zo1anA com.my.interface /my/path "value"
+```
+
+Please note, "value" is whatever your raw value is, no encapsulation whatsoever is required by astarte or astartectl (but you can provide your own, if you need it). 
+Exceptions follow:
+- arrays are written like ["value", "anothervalue"] (depending on your shell, this can also be "['value', 'anothervalue']", please check before using in a production env)
+- binaryblobs must be base64 encoded
+- datetime is flexible enough, and can translate a fairly high number of different formats, list follows
```

Comment:
If this list is from an external library we could link to that library. So this does not get out of date in case of changes in the underlying library.